.drone.yml

```diff
@@ -1,15 +1,15 @@
 pipeline:
-  build_4:
-    image: node:4.4.2
+  build_6:
+    image: node:6
     commands:
-      - npm --loglevel warn install -g npm@3
+      - npm --loglevel warn install
```

Comment:
removed the install of npm 3 because it doesn't seem relevant to node 6

Reply:
That line needs to be removed. Now you have `npm --loglevel warn install` running twice.